### PR TITLE
optimize node port-group check

### LINKED ISSUE #1488: High ovn request count for kube-ovn-controller
Similar to #1486, when nodes count increased from 3 to 8 and 13, ovn request count increased from 80 to 160 and 230 

image

### PR DESCRIPTION
**Pull request by hongzhen-ma**
#### What type of this PR
- Bug fixes
####
optimize node port-group check

#### Which issue(s) this PR fixes:
Fixes #1488

### CODE REVIEW

File: pkg/controller/controller.go [hunk 1]
```diff
@@ -728,7 +728,7 @@ func (c *Controller) startWorkers(stopCh <-chan struct{}) {
 	go wait.Until(c.CheckGatewayReady, 5*time.Second, stopCh)
 
 	if c.config.EnableNP {
-		go wait.Until(c.CheckNodePortGroup, 10*time.Second, stopCh)
+		go wait.Until(c.CheckNodePortGroup, time.Duration(c.config.NodePgProbeTime)*time.Minute, stopCh)
```
Comment (oilbeater) [inline pkg/controller/controller.go:731]: 5 minutes is too long for networkpolicy to take effect

**Review by oilbeater — CHANGES_REQUESTED** [review]

**Comment by oilbeater** [comment]
should add data comparsion to show the improvement

File: pkg/controller/controller.go (hunk 1 above)
Comment (hongzhen-ma) [reply pkg/controller/controller.go:731]: modified. Changed to 1 minute.

**Comment by hongzhen-ma** [comment]
wecom-temp-464b6984bd786dfd54c82faeee519045

分两次操作，每次添加5个假节点，在稳定后，ovn db的访问量，优化前为90 左右，优化后约30 左右

wecom-temp-adecbc1bd3cd01dd1077a016787c0b0d

**Review by oilbeater — APPROVED** [review]

**Comment by hongzhen-ma** [comment]
已回合release-1.9分支

**Pull request merged**